src/plugins/intel_cpu/src/emitters/snippets/x64/utils.cpp

```diff
@@ -46,8 +46,8 @@ size_t get_buffer_cluster_id(const ov::snippets::lowered::ExpressionPort& port)
 Xbyak::Reg64 get_aux_gpr(const std::vector<size_t>& used_gpr_idxs) {
     // RSP, RBP - stack-related registers, abi_param1 - runtime parameter register in the kernel
     static std::unordered_set<size_t> blacklist_gpr_idxs = {Xbyak::Operand::RSP,
-                                                            Xbyak::Operand::RBP,
-                                                            static_cast<size_t>(abi_param1.getIdx())};
+                                                            static_cast<size_t>(abi_param1.getIdx()),
+                                                            static_cast<size_t>(abi_param2.getIdx())};
```

Comment:
Could you please elaborate why we pass `abi_param2` now? The reason is case with static kernel or smth another?

Minor: could you please update the comment?